src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonProxy.cs

```diff
@@ -85,6 +114,7 @@ public static Props Props(string singletonManagerPath, ClusterSingletonProxySett
         private string _identityId;
         private IActorRef _singleton = null;
         private ICancelable _identityTimer = null;
+        private ICancelable _identityTimeoutTimer = null;
```

Comment:
Simply out of curiosity, why not the IWithTimers used instead of Context.System.Scheduler?

Reply:
Good point, I'll try and convert the class to use IWithTimers

Reply:
Done